### PR TITLE
[CI] Do not set Intel OMP for aarch64

### PR DESCRIPTION
**Pull request by desertfire**
As title

### CODE REVIEW

**Review by angelayi — APPROVED** [review]

File: .ci/pytorch/test.sh [hunk 1]
```diff
@@ -691,12 +691,14 @@ test_inductor_get_core_number() {
...
 test_inductor_set_cpu_affinity(){
   #set jemalloc
-  JEMALLOC_LIB="$(find /usr/lib -name libjemalloc.so.2)"
-  IOMP_LIB="$(dirname "$(which python)")/../lib/libiomp5.so"
-  export LD_PRELOAD="$JEMALLOC_LIB":"$IOMP_LIB":"$LD_PRELOAD"
-  export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:-1,muzzy_decay_ms:-1"
-  export KMP_AFFINITY=granularity=fine,compact,1,0
-  export KMP_BLOCKTIME=1
+  if [[ "${TEST_CONFIG}" != *aarch64* ]]; then
+    JEMALLOC_LIB="$(find /usr/lib -name libjemalloc.so.2)"
```
Comment (malfet) [inline .ci/pytorch/test.sh:695]: Can you please move JEMALLOC and MALLOC_CONFG outside  of that loop?

**Comment by malfet** [comment]
@desertfire any objections if I'll revert this one, as I think JEMALLOC changes help a lot, only IOMP stuff can be guarded (and it's benign, as there are no such library on aarch64 platform)

File: .ci/pytorch/test.sh (hunk 1 above)
Comment (desertfire) [reply .ci/pytorch/test.sh:695]: I wasn't sure if libjemalloc exists on aarch64 CI. I can add it back in another PR.

**Pull request closed without merging**